pom.xml

```diff
@@ -1063,6 +1076,9 @@
                 <groupId>org.jacoco</groupId>
                 <artifactId>jacoco-maven-plugin</artifactId>
                 <version>${jacoco-maven-plugin.version}</version>
+                <configuration>
+                    <excludes>**/*.jar</excludes>
+                </configuration>
```

Comment:
- This is to solve the problem https://github.com/apache/shardingsphere/actions/runs/4054751320/jobs/6976968248 encountered. Jacoco actively scans JARs.